server/app/mutations/grids/unassign_user.rb

```diff
@@ -24,7 +24,8 @@ def validate
 
     def execute
       user.grids.delete(grid)
+      user.save # relation callback has bug so we have to explicitly call update here
```

Comment:
Does this need to be `save`, vs an explicit `publish_update_event` like elsewhere in the code?